content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
+| {serial_lunid}K5GS1SJA_5000cca25e29ec8c     | sdh  | scsi      | 2160   | K5GS1SJA     | 5000cca25e29ec8c | 2000398934016 |             | Auto         | Always On  | Disabled     | true        |              | <null>     |        | <null>   | <null>     | <null>        | HUS726020AL4210           | 7200         | HDD  | <null>   | <null>   | SCSI | mymirror       | <null>                 | <empty list>     | <dict>    | <empty list> | sdh     |
+| {serial_lunid}K5H5G05A_5000cca25e4247bc     | sdi  | scsi      | 2176   | K5H5G05A     | 5000cca25e4247bc | 2000398934016 |             | Auto         | Always On  | Disabled     | true        |              | <null>     |        | <null>   | <null>     | <null>        | HUS726020AL4210           | 7200         | HDD  | <null>   | <null>   | SCSI | hotsparepool   | <null>                 | <empty list>     | <dict>    | <empty list> | sdi     |
+| {serial_lunid}N4G9LPHY_5000cca24511760c     | sdj  | scsi      | 2192   | N4G9LPHY     | 5000cca24511760c | 2000398934016 |             | Auto         | Always On  | Disabled     | true        |              | <null>     |        | <null>   | <null>     | <null>        | HUS726020AL4210           | 7200         | HDD  | <null>   | <null>   | SCSI | ecrypt         | <null>                 | <empty list>     | <dict>    | <empty list> | sdj     |
+| {serial_lunid}N4G2SZMY_5000cca245050b28     | sdk  | scsi      | 2208   | N4G2SZMY     | 5000cca245050b28 | 2000398934016 |             | Auto         | Always On  | Disabled     | true        |              | <null>     |        | <null>   | <null>     | <null>        | HUS726020AL4210           | 7200         | HDD  | <null>   | <null>   | SCSI | ecrypt         | <null>                 | <empty list>     | <dict>    | <empty list> | sdk     |
+| {serial_lunid}N4G22WKK_5000cca24503ce58     | sdl  | scsi      | 2224   | N4G22WKK     | 5000cca24503ce58 | 2000398934016 |             | Auto         | Always On  | Disabled     | true        |              | <null>     |        | <null>   | <null>     | <null>        | HUS726020AL4210           | 7200         | HDD  | <null>   | <null>   | SCSI | <null>         | <null>                 | <empty list>     | <dict>    | <empty list> | sdl     |
++---------------------------------------------+------+-----------+--------+--------------+------------------+---------------+-------------+--------------+------------+--------------+-------------+--------------+------------+--------+----------+------------+---------------+---------------------------+--------------+------+----------+----------+------+----------------+------------------------+------------------+-----------+--------------+---------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+{{< expand "Listing Unused Disks with Partitions" "v" >}}
+
+#### Description
+
+The `join_partitions` argument determines if the `get_unused` command returns all partitions written to each disk. The default is false.
```

Comment:
After this, tell them how to enter the argument. Also in this we should call it the property.
```suggestion
The `join_partitions` property argument determines if the `get_unused` command returns all partitions written to each disk. The default is false.
```